src/hearings/components/hearing-actual-summary/hearing-actual-summary.component.html

```diff
@@ -0,0 +1,120 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l">{{hearingActualsMainModel?.caseDetails.publicCaseName}}</span>
+      Hearing details
+    </h1>
+
+    <exui-listing-information-summary></exui-listing-information-summary>
```

Comment:
Could we put this <exui-listing-information-summary></exui-listing-information-summary> back to src/hearings/containers/view-hearing/hearing-completed-summary/hearing-completed-summary.component.html? As basic common component shouldn't be inside another basic common component.